packages/accounts-base/accounts_common.js

```diff
@@ -212,9 +213,14 @@ export class AccountsCommon {
             DEFAULT_LOGIN_EXPIRATION_DAYS) * 24 * 60 * 60 * 1000;
   }
 
-  _getPasswordResetTokenLifetimeMs() {
-   return (this._options.passwordResetTokenExpirationInDays ||
-            DEFAULT_PASSWORD_RESET_TOKEN_EXPIRATION_DAYS) * 24 * 60 * 60 * 1000;
+  _getPasswordResetTokenLifetimeMs(reason) {
```

Comment:
I don't see a reason not to break this down into two functions as well (`_getPasswordResetTokenLifetimeMs` and `_getPasswordEnrollTokenLifetimeMs`).

The name just isn't accurate otherwise.  (i.e. `_getPasswordResetTokenLifetimeMs` is really `_getPasswordResetOrEnrollTokenLifetimeMs` which is just too long).